### PR TITLE
FAT-4029: greening of autotest

### PR DESCRIPTION
**Pull request by re-jas**
https://issues.folio.org/browse/FAT-4029

### CODE REVIEW

File: cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js [hunk 1]
```diff
@@ -78,8 +76,9 @@ describe('ui-circulation-settings: Fixed due date schedules', () => {
           departments: [],
         })
           .then((user) => {
-            userId = user.id;
-            userBarcode = user.barcode;
+            userData.userId = user.id;
+            userData.userBarcode = user.barcode;
+            userData.userLastName = user.lastName;
```
Comment (MaksimTerekhov) [inline cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js:81]: you can use spread operator
`userData = [ ...user ]`

File: cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js [hunk 2]
```diff
@@ -167,7 +166,7 @@ describe('ui-circulation-settings: Fixed due date schedules', () => {
       checkInDate: moment.utc().format(),
     })
       .then(() => {
-        Users.deleteViaApi(userId);
+        Users.deleteViaApi(userData.userId);
```
Comment (MaksimTerekhov) [inline cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js:169]: Seems a lot of 'user' word here
`userData.userId`
let's use just userData.id

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js]: done

File: cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.cy.js]: done

**Review by TetianaParanich — APPROVED** [review]

**Review by penfriend — APPROVED** [review]

**Pull request merged**